ob-yaml.md

```diff
@@ -261,6 +263,8 @@ orders:
         vault-id: 0xabcd
 ```
 
+> **Note:** Tokens referenced in `inputs` or `outputs` do not need to be predefined in the top-level `tokens` mapping for the YAML file to be valid. However, before deployment, the application must resolve token details (network, address, decimals) either from the `tokens` mapping or by querying the contract on-chain (for example, via the SDK).
```

Comment:
shouldn't this comment about resolving token details be a note for the tokens section of the spec?

from the perspective of orders, i'm not sure what this is saying exactly